src/overlays/actors/ovl_En_Ma3/z_en_ma3.c

```diff
@@ -79,9 +82,6 @@ static UNK_PTR D_80AA38B0[] = {
     0x06003070,
 };
 
-extern u32 D_06005420;
-extern FlexSkeletonHeader D_06008D90;
-extern AnimationHeader D_060093BC;
 extern AnimationHeader D_06009EE0;
```

Comment:
Missed this one? Looks like there are a couple of animations in the blob in `object_ma2`

Reply:
yeah they're in a blob, I haven't messed with any xmls yet

Reply:
There's some segment 6 stuff here too

Reply:
this pr wasn't really intended to actually fix symbols, necessarily, it was just supposed to add the auto gen'd object symbols